src/nix/modules/service/docker-compose-service.nix

```diff
@@ -90,6 +90,11 @@ in
       default = null;
       description = dockerComposeRef "command";
     };
+    service.container_name = mkOption {
```

Comment:
This seems to be unused.
Please add it to `config.build.service` near the bottom.
Adding a good [test case](https://github.com/hercules-ci/arion/blob/bb54a451b1a66ddc9a374d5dedd8c26b8bf18bef/tests/arion-test/default.nix) prevents this kind of error.

Reply:
fixed

> Please add it to config.build.service near the bottom.

not near bottom, but near `command`